workers/ohsome_quality_analyst/api/api.py

```diff
@@ -235,6 +236,16 @@ async def get_available_regions():
     return response
 
 
+@app.get("/indicatorLayerCombinations")
+async def list_indicator_layer_combinations(
+    request: Request,
+):
+    """List names of available indicator-layer-combinations."""
+    response = empty_api_response(request.url._url)
```

Comment:
```suggestion
async def list_indicator_layer_combinations():
    """List names of available indicator-layer-combinations."""
    response = empty_api_response()
```

The url parameter is removed.